manageiq-operator/api/v1alpha1/helpers/miq-components/kafka.go

```diff
@@ -228,7 +240,51 @@ func KafkaDeployment(cr *miqv1alpha1.ManageIQ, scheme *runtime.Scheme) (*appsv1.
 		},
 		Env: []corev1.EnvVar{
 			corev1.EnvVar{
-				Name: "KAFKA_BROKER_USER",
+				Name:  "KAFKA_ENABLE_KRAFT",
+				Value: "yes",
+			},
+			corev1.EnvVar{
+				Name:  "KAFKA_CFG_PROCESS_ROLES",
+				Value: "broker,controller",
+			},
+			corev1.EnvVar{
+				Name:  "KAFKA_CFG_CONTROLLER_LISTENER_NAMES",
+				Value: "CONTROLLER",
+			},
+			corev1.EnvVar{
+				Name:  "KAFKA_CFG_LISTENERS",
+				Value: "INTERNAL://:9092,CONTROLLER://:9093",
+			},
+			corev1.EnvVar{
+				Name:  "KAFKA_CFG_INTER_BROKER_LISTENER_NAME",
+				Value: "INTERNAL",
+			},
+			corev1.EnvVar{
+				Name:  "KAFKA_CFG_ADVERTISED_LISTENERS",
+				Value: "INTERNAL://kafka:9092",
+			},
+			corev1.EnvVar{
+				Name:  "KAFKA_CFG_CONTROLLER_QUORUM_VOTERS",
+				Value: "1@kafka:9093",
+			},
+			corev1.EnvVar{
+				Name:  "KAFKA_CFG_BROKER_ID",
+				Value: "1",
+			},
+			corev1.EnvVar{
+				Name:  "KAFKA_CFG_OFFSETS_TOPIC_NUM_PARTITIONS",
+				Value: "25",
+			},
```

Comment:
See discussion in https://github.com/ManageIQ/manageiq-pods/pull/936. If we move ahead with this partition limit then https://github.com/ManageIQ/manageiq-pods/pull/936 can be closed.

Reply:
I wonder if we should expose this from the CR?  cc @agrare ?

Reply:
I've removed this since the partition limit is no longer necessary as https://github.com/ManageIQ/manageiq-pods/pull/936 is now merged